### DISCUSSION TITLE
Storage location for home icons

### DISCUSSION THREAD

**Discussion by bostrump** [jcorporation/myMPD#1259] [Q&A]
My question relates to [myMPDos](https://github.com/jcorporation/myMPDos), but there is no discussion area for this project.

Who can tell me how the home icons are persisted?
I would have expected home icons to be saved in a status file ("home_list"?) in "/var/lib/mympd/state".
After creating an icon, I can't see any changes there.
However, the new home icon does appear on another end device when I open the myMPD web interface there.
So the home icon must be persisted somewhere, but where?

Why I'm asking:
After reebot using the myMPDos script savereboot.sh, all home icons are lost.
So they are obviously not captured by the Alpine Linux "lbu commit".

I don't know if there is a correlation, but it is noticeable that after reebot is finished the web interface can no longer connect to myMPD. 
Error mesage: "API Fehler" "Fehler beim Zugriff auf /api/default"

Only after reloading the page does the connection work again (but without home icons)
Other settings are saved correctly (e.g. background color)

I created the myMPDos image on 17.04.2024 (using myMPD 14.1.2) and it is running on Raspberry Pi 4 B.

**Comment by jcorporation** [comment]
myMPD saves the home icons on stop and on the SIGUP signal. I tested it shortly with current release and it works perfectly for me. Seems to by a myMPDos specific issue.
